apps/src/templates/census2017/CensusInaccuracyReview.jsx

```diff
@@ -0,0 +1,400 @@
+import React, { PropTypes, Component } from 'react';
+import {Table} from 'reactabular';
+import Button from '../Button';
+
+export default class CensusInaccuracyReview extends Component {
+  static propTypes = {
+    authenticityToken: PropTypes.string,
+    reportsToReview: PropTypes.array,
+  };
+
+  state = {
+    toReview: undefined,
+    notes: '',
+    resolvedReports: [],
+  };
+
+  formatTeachesCs = (teachesCs) => {
+    return (
+      <div style={{textAlign: 'center', verticalAlign: 'middle'}}>
+        {teachesCs ? teachesCs : "Null"}
+      </div>
+    );
+  };
+
+  formatComment = (comment) => {
+    return (
+      <div>
+        {comment}
+      </div>
+    );
+  };
+
+  mapHistoricSummary = (teaches_cs) => {
+    switch (teaches_cs) {
+      case null:
+        return 'Null';
+      case undefined:
+        return 'No Data';
```

Comment:
Is there a meaningful difference here between null and no data? If so, can we use more descriptive words, esp for the null case?

Reply:
The difference is that `teaches_cs` could legitimately have a value of `null` but if it is `undefined` it indicates that we didn't have any value at all. This occurs because we are pulling the input out of an array which may be shorter than the number of years back we look. Specifically, right now we generate summaries for 2016 and 2017 but for each of those we look back both one and two years. So, for 2017 summaries the two-year-old data will be `undefined` because we have "No Data" for that year's summary while the one-year-old data can have a summary but with `teaches_cs` set to `null`.

I think this will be clearer if I change it from "No Data" to "No Summary for this Year"

Reply:
Yes, that would be much more clear. I'm not sure who will be consuming this UI, but `null` and `no data` seem very technical to me, and don't reveal any context. Thx

Reply:
Never mind. This code isn't actually used. It was leftover from a refactor. I can just remove it.